homeassistant/components/hassio/ingress.py

```diff
@@ -109,6 +109,9 @@ async def _handle(
     delete = _handle
     patch = _handle
     options = _handle
+    connect = _handle
+    head = _handle
+    trace = _handle
```

Comment:
`TRACE` seems to be useful toghether with `Max-Forwards`, which says:

> The HTTP Max-Forwards [request header](https://developer.mozilla.org/en-US/docs/Glossary/Request_header) is used with the [TRACE](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/TRACE) method to limit the number of nodes (usually [proxies](https://developer.mozilla.org/en-US/docs/Glossary/Proxy_server)) that the request goes through.

It doesn't look like we implement `Max-Forwards`. But then, I am also not sure how `TRACE` is usually used, and maybe it is useful as is, without handling `Max-Forwards`.